modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -301,6 +301,11 @@ If you opted to use private API endpoints, you must use an existing VPC and Priv
 ====
 You must ensure that your VPC is configured with a public and a private subnet for each availability zone that you want the cluster installed into. If you opted to use PrivateLink, only private subnets are required.
 ====
+.. Optional: Expand *Additional security groups* and select additional custom security groups to apply to nodes in the machine pools created by default. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.
```

Comment:
@laubai Hi, Security group changes as part of day1 actions from OCM UI (ROSA) not yet ready for production release. I'm not sure if it is good to add them as part of day 2 changes which is already in production behind the feature flag.
Regarding doc changes on day 1, It seems like wrong because we support adding security group for all node types including control plane, infrastructure, worker nodes.
![image](https://github.com/openshift/openshift-docs/assets/93653094/ce398dac-35f4-4407-b15d-0b2b70e14ed8)
  Suggestion is
`Optional: Expand *Additional security groups* and select additional custom security groups to apply to all nodes types such as control plane, infrastructure, worker created by default. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.` 
Please review it.

Reply:
Thanks @jmekkatt - based on talking with @celdrake yesterday, this update will not be ready until after our planned publish date for this.